uisdk/src/androidTest/java/com/karhoo/uisdk/booking/braintree/BraintreeBookingTests.kt

```diff
@@ -107,7 +107,8 @@ class BraintreeBookingTests : Launch {
         booking(this, null) {
             mediumSleep()
         } result {
-            checkSnackbarWithText(R.string.K0001)
+            // R.string.K0001
```

Comment:
Does the comment need to be there? Same with the other test for K0001

Reply:
I thought to add the resource ID to help the reader understand where the message come from

Reply:
I can remove it if you think it is unnecessary.